pytket/extensions/qujax/qujax_convert.py

```diff
@@ -36,9 +39,79 @@ def _tk_qubits_to_inds(tk_qubits: Sequence[Qubit]) -> Tuple[int, ...]:
     return tuple(q.index[0] for q in tk_qubits)
 
 
+def _append_func(f: Callable, func_to_append: Callable) -> Callable:
```

Comment:
Functions with names starting with `_` will not be shown in the documentation, is that intended?

Reply:
yep!